### DISCUSSION TITLE
请问这个uiautomation要怎么使用。没用相关的功能所以不太了解。

### DISCUSSION THREAD

**Discussion by DarksKnight** [netease-im/NIM_Duilib_Framework#373] [General]
请问这个uiautomation要怎么使用。没用相关的功能所以不太了解。

**Comment by nmgwddj** [comment]
首先要问的是你需要什么功能？为什么要用 UIAutomation 能力？

**Reply by DarksKnight** [reply]
软件功能比较多，需要进行自动化测试，方便质检验收。

**Comment by nmgwddj** [comment]
可以了解一下这个包：https://github.com/yinkaisheng/Python-UIAutomation-for-Windows/blob/master/readme_cn.md

**Reply by DarksKnight** [reply]
好的，我看看
